templates/cli/lib/commands/push.js.twig

```diff
@@ -1023,14 +1024,14 @@ const pushTeam = async ({ all, yes } = {}) => {
     }
 }
 
-const pushMessagingTopic = async ({ all, yes } = {}) => {
+const pushMessagingTopic = async () => {
     let response = {};
 
     let topicsIds = [];
     const configTopics = localConfig.getMessagingTopics();
-    let overrideExisting = yes;
+    let overrideExisting = cliConfig.force;
 
-    if (all) {
+    if (cliConfig.all) {
```

Comment:
As part of this PR, can we also simplify all actions? For example here, this is the flow:

- If all, push all from cofig to local array
- If not all, ask question, then set answer to local array

What I was thinking instead:
- Ask question

The logic for `all` or `force` or others will all live inside `questions.js`. There, we could wrap question in functions like `allowAll`, `allowForce`.. Let's imagine in future we have this like of questions:

- What is your name?
- Select gender
- Want to save profile?

and I ran question with `--yes --id="male`, then I should be asked name, and that's it. Instead of asking me rest of questions, it can pre-fill values  from tags. Also, if all questions are automated, it should not ask at all - pre-fill and submit everything.